tasks/lib/spidey_stuff.rb

```diff
@@ -0,0 +1,29 @@
+require_relative 'parse_stuff'
+
+class SlackApiSpider < Spidey::AbstractSpider
+  include ParseStuff
```

Comment:
This is OK with me, although I tend to avoid names like `Stuff` :) Since `ParseStuff` is not reused, I would just put the methods into `SlackApiSpider`. Also minor, `SlackApiSpider` is maybe more suitable to be called `SlackApiDocumentationSpider`.